.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,7 +22,8 @@ labels: bug
 ## Environment
 
 - [ ] PostHog Cloud
-- [ ] self-hosted PostHog, version/commit: _please provide_
+- [ ] self-hosted PostHog (ClickHouse), version/commit: _please provide_
+- [ ] self-hosted PostHog (Postgres), version/commit: _please provide_
```

Comment:
I'd try somehow to set expectations such that people don't expect us to be fixing Postgres bugs super quickly.

Maybe something like:
```
- [ ] self-hosted PostHog (Postgres - Note this is deprecated), version/commit: _please provide_
```

Reply:
`is deprecated` at this point feels a bit strong, maybe we can say `- Note this will be deprecated soon`?

Reply:
Maybe `(Postgres – legacy)`?